src/Microsoft.AspNetCore.Blazor.Browser/Rendering/BrowserRenderer.cs

```diff
@@ -86,20 +87,20 @@ public void Dispose()
         }
 
         /// <inheritdoc />
-        protected override void UpdateDisplay(in RenderBatch batch)
+        protected override Task UpdateDisplay(in RenderBatch batch)
```

Comment:
I know I commented on this in another PR - but we should eventually make a decision about whether we're using `Async` on async methods. We've been making lifecycle methods async as the framework matures. This is a good topic for later I suppose.

Reply:
Good point. Renamed to `UpdateDisplayAsync`.